scene/gui/item_list.cpp

```diff
@@ -1853,6 +1866,7 @@ ItemList::ItemList() {
 	add_child(scroll_bar, false, INTERNAL_MODE_FRONT);
 
 	scroll_bar->connect("value_changed", callable_mp(this, &ItemList::_scroll_changed));
+	connect("mouse_exited", callable_mp(this, &ItemList::_mouse_exited));
```

Comment:
```suggestion
	scroll_bar->connect("value_changed", callable_mp(this, &ItemList::_scroll_changed));

	connect("mouse_exited", callable_mp(this, &ItemList::_mouse_exited));
```

This new line is unrelated to the scrollbar, so it shouldn't be visually grouped with it. And scrollbar's own connect method can be moved next to its related code above.